7.0 Update Canada account chart and taxes

dufresnedavid opened the pull request:
Add the required account for the Canada payroll. In l10n_ca_hr_payroll_account, the salary rules in the demo data are linked to these accounts.
Set the refund sign of every tax in Canada to -1.

dufresnedavid commented:
@max3903 I fixed both the account chart and the tax chart.

max3903 commented:
:+1:

odony commented:
Hi,

It would really help reviews if you avoid reindenting large blocks of code, because it adds a lot of noise in the PR. If you really need to reindent, could you make that a separate PR please?

Is there any reason why you are renumbering existing accounts instead of giving the new ones new codes and XML IDs? Some of these changes seem pretty major, for example "21412 Federal Income Tax" becomes a new view account "CANADA PENSION PLAN TO PAY". That sounds like a bad idea.

While you're fixing typos, you could probably fix "PROVINCIAL REVENU AGENCY" too.

The rest looks good to me, I assume this has been validated with other Canadian partners/customers?

Thanks,

dufresnedavid commented:
@odony I removed the indentation of code from the PR. This pull request only modifies accounts related to the payroll. Since no serious implementation of the Canada Payroll has been published already, I changed a few accounts to make the chart more logical.

Thanks for the review

mart-e commented:
Thank you, I have merged the taxes at f3339c6 and the chart of account at e601a41.
For the future, please try to keep the diff minimal.
